### PR TITLE
Updated playback sync examples and added summary calls for movies and shows

### PR DESCRIPTION
**Pull request by Razzeee**
I've also just added summary calls for movies and shows.
We probably also need an implementation for the extended Info part.

The naming and how I did this is also up for discussion, as I'm not sure you would have done it like this @fuzeman

### CODE REVIEW

**Comment by Razzeee** [comment]
Also see https://github.com/fuzeman/trakt.py/issues/20
@tknorris

**Comment by fuzeman** [comment]
Merged

------------------------

**Notes:**
 - The movies/shows (summary) interfaces return parsed objects now (like all the other interfaces), instead of the raw response from trakt.tv
 - The ```Episode.to_dict()``` method doesn't return the ```title``` field yet
 - ```Episode.pk``` has changed to a tuple: ```(<season>, <episode>)```
 - *still testing a number of things, there might be a few bugs*

**Comment by Razzeee** [comment]
Just a remark as you said 'like all the other interfaces' 
This isn't true for scrobble, as far as I remember

**Comment by tknorris** [comment]
So the only two issues I've found so far are both in episode summaries.
1. As you already mentioned, title doesn't come through for Episode.to_dict(). I need it for the ratings window.
2. The episode ids don't seem to be coming through at all. Every episode I test has a an empty id's dictionary. I need those in order to be able to submit an episode rating.

Otherwise, I've got manual ratings working perfectly for Shows and Movies.

Razzeee, does that change above where Episode.pk is now a tuple cause other breakage for you? I haven't tested any of the other areas of script.trakt with the new script.module.trakt code from fuzeman.

**Comment by Razzeee** [comment]
Haven't tested anything yet.
I would think it's not breaking anything as I think we're using other means of getting to the episode level (because I'm a python noob that hasn't understood tuple pk's prior to the last checkins from fuzeman)

But I will try to check later.

**Comment by fuzeman** [comment]
@Razzeee by "other interfaces" I was meaning the get methods on the **/sync** interfaces. I might look at returning parsed objects from other interfaces/methods eventually, but this would break anything expecting a ```dict``` to be returned..

---------------

@tknorris I've just committed a number of changes to the develop branch:
 - ```Episode.to_dict()``` "number" field doesn't return a tuple anymore
 - ```Episode.to_dict()``` "title" field is now included
 - ```Episode``` and ```Season``` "ids" field is now returned correctly from `to_dict()`

**Comment by tknorris** [comment]
Thanks for the quick fix! Seems to work great!

You might want to consider including a "season" key in the episode dict. I happened to have it available already so I was able to work around it not being there but I can imagine it would be really useful to others.

Thanks again!

**Comment by s7eele** [comment]
great news, thx for all your efforts

**Comment by fuzeman** [comment]
@tknorris yeah, was thinking about adding a "season" field. I'll probably add this before the next master release.

File: examples/collection.py [hunk 1]
```diff
@@ -61,6 +55,19 @@
         print '\t', 'plays', '\t' * 3, movie.plays
 
         print
+		
+	    # Fetch playback for shows
+    playback = Trakt['sync/playback'].shows(exceptions=True)
```
Comment (Razzeee) [inline examples/collection.py]: This needs an update as this is no longer working

File: examples/collection.py [hunk 2]
```diff
@@ -61,6 +55,19 @@
...
+
+    for key, item in playback.items():
+        print item
+
+        if type(item) is Show:
```
Comment (Razzeee) [inline examples/collection.py]: Is there something like an Episode type?

**Pull request merged**